sources/api/thar-be-updates/src/status.rs

```diff
@@ -100,11 +100,9 @@ pub fn get_update_status(_lockfile: &File) -> Result<UpdateStatus> {
     let status_file = File::open(UPDATE_STATUS_FILE).context(error::NoStatusFileSnafu {
         path: UPDATE_STATUS_FILE,
     })?;
-    Ok(
-        serde_json::from_reader(status_file).context(error::StatusParseSnafu {
-            path: UPDATE_STATUS_FILE,
-        })?,
-    )
+    serde_json::from_reader(status_file).context(error::StatusParseSnafu {
+        path: UPDATE_STATUS_FILE,
+    })
```

Comment:
How is the `Result` returned without the `Ok()`?

Reply:
`.context` converts the `Result<JsonError` to `Result<OutError>` which is being returned since we are not using `?` on it.

Reply:
Thanks!